.github/workflows/release.yaml

```diff
@@ -57,25 +57,27 @@ jobs:
               uses: stefanzweifel/git-auto-commit-action@v4
               with:
                 file_pattern: 'docs'
-                commit_message: "Changes in docs for release: v${{ github.event.inputs.release-version }}"
-            - name: Create Github release
-              uses: ncipollo/release-action@v1
-              with:
-                  tag: "v${{ github.event.inputs.release-version }}"
+                commit_message: "Changes in docs for release: ${{ github.event.inputs.release-version }}"
 
             - name: Image Build
               run: |
                 cd custom-nb-image
-                docker build --build-arg SDK_VERSION="${{ github.event.inputs.release-version }}" -t quay.io/${{ github.event.inputs.quay-organization }}/notebook:v${{ github.event.inputs.release-version }} .
-                docker tag quay.io/${{ github.event.inputs.quay-organization }}/notebook:v${{ github.event.inputs.release-version }} quay.io/${{ github.event.inputs.quay-organization }}/notebook:latest
+                podman build --build-arg SDK_VERSION="${{ github.event.inputs.release-version }}" -t quay.io/${{ github.event.inputs.quay-organization }}/notebook:${{ github.event.inputs.release-version }} .
```

Comment:
The SDK_VERSION can not have `v0.5.0` with `v` prefix or the step fail with `ERROR: Could not find a version that satisfies the requirement codeflare-sdk==v0.0.5`